docs/api/process.md

```diff
@@ -111,6 +111,12 @@ The `process` object has the following methods:
 
 Causes the main thread of the current process crash.
 
+### `process.getCreationTime()`
+
+Returns `number`
+Indicates the creation time of the application.
+The time is represented as number of milliseconds since epoch. It returns null if it is unable to get the process creation time.
+
```

Comment:
I _think_ we need to document this as "Returns `Menu | null`" for typescript. Something like

```text
Returns `Menu | null` - The number of milliseconds since epoch, or `null` if the information is unavailable
```

Reply:
done @ckerr

Reply:
@ckerr capitalized it.